src/main/scala/org/globalnames/parser/SciName.scala

```diff
@@ -11,15 +11,15 @@ import org.apache.commons.lang.StringEscapeUtils
 import scala.util.matching.Regex
 
 case class SciName(
-  verbatim: String,
-  normalized: Option[String] = None,
-  canonical: Option[String] = None,
-  isParsed: Boolean = false,
+  verbatim: String = "",
+  ast: Option[NamesGroup] = None,
   isVirus: Boolean = false,
   isHybrid: Boolean = false,
-  parserRun: Int = -1,
-  parserVersion: String = BuildInfo.version
+  isParsed: Boolean = false,
+  canonical: Option[String] = None,
+  normalized: Option[String] = None
 ) {
+  val parserVersion: String = BuildInfo.version
```

Comment:
I doubt `parserVersion` should be here. It is exactly same for all `SciName`s parsed within single run.